index.html

```diff
@@ -125,7 +125,7 @@ <h2 id="MySteps" tabindex="-1">
                 </div>
             </div>
             <div id="EditorArea">
-                <div class="myWorkArea">
+                <div class="myWorkArea" style="min-width: 800px;">
```

Comment:
there is a class defined in style.css. Please move it.

Reply:
ok